### DISCUSSION TITLE
FullScreen Rectangle Op | Option Scale: Set to Fit needs to be triggered in runtime - Doesn't work by default

### DISCUSSION THREAD

**Discussion by jaalibandar** [cables-gl/cables_docs#471] [! Welcome - readme first!]
Here is a link to the test patch: https://cables.gl/edit/RPXrEn 

Observation: Once patch it loaded, 1. Look at the viewport - It is fully grey - 2. Check the FullscreenRectangle Op - Scales is set to Fit by default - 3. Change the scale toggle to Stretch and back to Fit - The grey rectangle becomes a square now(Expected output that should be observed when patch loads without the tinkering) | Same on Dev

Expected View when the patch loads
![image](https://user-images.githubusercontent.com/86233755/214541183-6d3c7df2-ed0d-4cdf-927f-81b2ef51f580.png)
 
Currently what happens when the patch load
![image](https://user-images.githubusercontent.com/86233755/214541319-f7bd34fa-bf45-498d-b340-07da0dea3b64.png)

Workaround: 
![image](https://user-images.githubusercontent.com/86233755/214541529-640cfae1-8257-4473-a8b9-df750a86bead.png)

**Comment by pandrr** [comment]
thanks for rewporting and example!

fixed on dev.cables.gl !

**Reply by jaalibandar** [reply]
Thanks!
